server/conf/routes

```diff
@@ -22,3 +22,5 @@ POST    /register                   controllers.UserController.postRegister
 #Login
 GET     /login                      controllers.UserController.getLogin
 POST    /login                      controllers.UserController.postLogin
+GET     /githublogin                controllers.GitHubUser.getLogin
+GET     /githubback                 controllers.GitHubUser.githubCallback(code: String)
```

Comment:
Right, I forget to fix this hole. Basically we need a state on server side to validate incoming "state"